test/unit/command/test_converge.py

```diff
@@ -38,3 +41,28 @@ def test_execute(mocker, patched_logger_info, patched_ansible_converge,
     patched_ansible_converge.assert_called_once_with()
 
     assert config_instance.state.converged
+
+
+def test_ansible_args(mocker, patched_logger_info, patched_ansible_converge,
```

Comment:
Perhaps name it `test_ansible_args_correctly_passed` to give more the "why" without the need then for the following comment?